src/components/ProfileOverview/index.js

```diff
@@ -3,7 +3,7 @@ const ProfileOverview = ({
                                  {username: "Dummy", firstName: "Harry", lastName: "Potter", phone: ""}
                          }) => {
     return (
-        <div>
+        <div style={{marginTop: '20px'}}>
```

Comment:
Can we use Bootstrap classes to add the margin? Like ```mt-2```?

Reply:
Fixed the problem. Thanks.